packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -284,7 +284,13 @@ export default defineComponent({
   },
   computed: {
     ...mapGetters('runtime/spaces', ['spaces']),
-    ...mapGetters('Files', ['versions', 'sharesTree', 'sharesTreeLoading']),
+    ...mapGetters('Files', [
+      'ancestorMetaData',
+      'currentFolder',
```

Comment:
`currentFolder` is never used. Seems to me like you don't need to introduce it here.